CMakeLists.txt

```diff
@@ -5,8 +5,16 @@ set(CMAKE_CXX_STANDARD_REQUIRED on)
 
 include(${CMAKE_CURRENT_SOURCE_DIR}/cmake/mason.cmake)
 
+option(WERROR "Add -Werror flag to build (turns warnings into errors)" ON)
+
+# Note: -D_GLIBCXX_USE_CXX11_ABI=0 is needed to support mason packages that are precompiled libs
+# Currently we only depend on a header only library, but this will help avoid issues when more libs are added via mason
 set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall -Wextra -pedantic -Wsign-compare -Wconversion -Wshadow")
```

Comment:
Looks as if you grabbed the comments about `-D_GLIBCXX_USE_CXX11_ABI=0` without the actual flag being added to the `CMAKE_CXX_FLAGS`? Best to pull that over too.